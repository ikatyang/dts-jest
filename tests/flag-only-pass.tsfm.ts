export {};

declare const string_to_any: (v: string) => any;

// @dts-jest
string_to_any('a');

// @dts-jest:only:pass:snapshot
string_to_any('b');

// @dts-jest
string_to_any('c');
