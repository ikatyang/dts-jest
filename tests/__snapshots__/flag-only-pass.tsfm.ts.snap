// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`string_to_any('b') 1`] = `"any"`;
